src/GraphQL/Utilities/SchemaPrinter.cs

```diff
@@ -324,11 +326,11 @@ public string FormatDefaultValue(object value, IGraphType graphType)
         {
             return graphType switch
             {
-                NonNullGraphType nonNull => FormatDefaultValue(value, nonNull.ResolvedType),
-                ListGraphType list => "[{0}]".ToFormat(string.Join(", ", ((IEnumerable<object>)value).Select(i => FormatDefaultValue(i, list.ResolvedType)))),
+                NonNullGraphType nonNull => FormatDefaultValue(value, nonNull.ResolvedType!),
+                ListGraphType list => "[{0}]".ToFormat(string.Join(", ", ((IEnumerable<object>)value).Select(i => FormatDefaultValue(i, list.ResolvedType!)))),
                 IInputObjectGraphType input => FormatInputObjectValue(value, input),
-                EnumerationGraphType enumeration => (enumeration.Serialize(value) ?? throw new ArgumentOutOfRangeException(nameof(value), $"Unknown value '{value}' for enumeration '{enumeration.Name}'")).ToString(),
-                ScalarGraphType scalar => AstPrinter.Print(scalar.ToAST(value)),
+                EnumerationGraphType enumeration => AstPrinter.Print(enumeration.ToAST(value) ?? throw new ArgumentOutOfRangeException(nameof(value), $"Unable to convert '{value}' to AST for enumeration type '{enumeration.Name}'.")),
```

Comment:
bugfix?

Reply:
Yes. Pretty obvious, considering the new scalar design.